include/DGContext2D.hpp

```diff
@@ -22,6 +22,8 @@ namespace blitzdg {
         real_matrix_type* yGrid_;
         real_matrix_type* Fscale_;
         index_matrix_type* Fmask_;
+        std::vector<index_type>* Gather_;
```

Comment:
I assume you are storing everything here by raw pointer since `DGContext2D` is non-owning of these resources?

Reply:
Yessir